### ISSUE TITLE
NTR: MRI: cranial nerve III small or hypoplastic (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8935]
New term request MRI: cranial nerve III small or hypoplastic (elthat_101022011655): Oculomotor; third; 3rd  UMLS: 5393522

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000597
- label: Third cranial nerve hypoplasia
- parents: [Abnormal cranial nerve morphology (HP:0001291)](https://hpo.jax.org/app/browse/term/HP:0001291)
- pmid: 
- def: Underdevelopment of the third cranial nerve (oculomotor nerve).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8935
- synonyms:
